.travis.yml

```diff
@@ -53,10 +53,11 @@ env:
   - SYSTEM=google:fedora-29-64 VARIANT=amd64
   - SYSTEM=google:fedora-30-64 VARIANT=amd64
 
-matrix:
-  allow_failures:
-    - env: SYSTEM=google:fedora-rawhide-64 VARIANT=amd64
```

Comment:
Maybe just comment this out, then?

Reply:
I don't think it is a syntax error "All checks have passed"

Reply:
I meant this would be:
```yaml
allow_failures:
#  - env: SYSTEM=google:fedora-rawhide-64 VARIANT=amd64
fast_finish: true
```

That's apparently valid YAML, not sure if Travis will like it, though.

Still, worth a try - commenting out just the rawhide item. I'd drop the `# Commented out…` line, too.